index.php

```diff
@@ -4,16 +4,78 @@
 <head>
   <title>13th Annual Alumni Meet &middot; Home</title>
   <?php include 'topHead.php'; ?>
+ <style>
+  .logo{margin:15px;}
+	h2
+	{	
+		margin-top:-80px;margin-left:-50px;
+	}
+ 
+ .item{margin-left:155px;}
+ <link href="css/carousel.css" rel="stylesheet" type="text/css">
+
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js"></script>
+  <script src="http://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/js/bootstrap.min.js"></script>
+
+ 
+ </style>
 </head>
 
 <body>
   <?php include "navbar.php"; ?>
-  <!-- <hr> -->
+  
 
   <div class="container">
     <div class="col-md-6" id="verticalLine">
       <img src="img/logo-square.jpg" alt="" class="logo" />
+   
+  
+   
+<div id="myCarousel" class="carousel carousel-fade" data-ride="carousel"><h2 align="center"><b>Sponsored by</b></h2>
+ 
+ 
+  <div class="carousel-inner" role="listbox">
+    <div class="item active">
+      <img class="img-responsive" src="img/spons/frontpage-sponsors/e3.png"width="148" height="148">
+    </div>
+  <div class="item">
+      <img class="img-responsive" src="img/spons/frontpage-sponsors/e2.png"width="148" height="148">
+    </div>
+    <div class="item">
+      <img class="img-responsive" src="img/spons/frontpage-sponsors/g11.png" width="250" height="300px">
+    </div>
+    <div class="item">
+      <img class="img-responsive"src="img/spons/frontpage-sponsors/e5.png"width="148" height="148">
+    </div>
+  <div class="item">
+      <img class="img-responsive"src="img/spons/frontpage-sponsors/luxuryhotelpartner.png"width="200" height="200">
+    </div>
+  <div class="item">
+      <img class="img-responsive"src="img/spons/frontpage-sponsors/g8.png"width="148" height="148">
     </div>
+  <div class="item">
+      <img src="img/spons/frontpage-sponsors/strategic.png"width="148" height="180">
+    </div>
+  <div class="item">
+      <img src="img/spons/frontpage-sponsors/g9.png"width="148" height="148">
+    </div>
+  <div class="item">
+      <img class="img-responsive"src="img/spons/frontpage-sponsors/e7.png"width="200" height="400">
```

Comment:
@punit121

Add a space before the SRC attribute. and also before the width attribute.